dessia_common/core.py

```diff
@@ -607,10 +608,50 @@ def to_xlsx_stream(self, stream):
         writer = XLSXWriter(self)
         writer.save_to_stream(stream)
 
+    def zip_settings(self):
+        """ Returns a list of streams containing different representations of the object. """
+        excel_stream = dcf.BinaryFile("excel_export.xlsx")
+        self.to_xlsx_stream(excel_stream)
+
+        json_stream = dcf.JsonFile("json_export.json")
+        self.save_to_stream(json_stream)
+
+        list_stream = [excel_stream, json_stream]
+        return list_stream
+
+    def to_zip_stream(self, archive: dcf.BinaryFile) -> List[dcf.BinaryFile]:
+        """ Creates a zip archive containing several files representing the export of DessiaObject. """
+        list_stream = self.zip_settings()
```

Comment:
Avoid setting the type of the variable in its name. dc's conventions state that we should name lists with an "s" at the end.

Here, for example `streams`